examples/actors/check_target.yaml

```diff
@@ -28,7 +28,7 @@ executor:
             targetinfo['docker'] = ('ERROR', msg)
             break
 
-    targetinfo['rsync'] = ('OK', None)
+    targetinfo['rsync'] = ('OK', '')
```

Comment:
Well if None is allowed this would have to be specified properly in the schema instead of working it around here. This is just one case, but there are most likely more around

Instead of 
```python:
    rsync = jsl.ArrayField(jsl.StringField(), max_items=2)
```
it should look like this:

```python:
    rsync = jsl.ArrayField(jsl.StringField(), jsl.OneOfField([jsl.StringField(), jsl.NullField()]))
```
Also looking at the containers key, I think this is wrong
@artmello I think you should check the error case on line: 39 in here

Reply:
I thought that the schema determines what is allowed. If it is decided that the code is authoritative and the schema should be adapted, I can adapt it as you suggest.

Reply:
The question is if the Schema is correct - To be quite honest I am not too happy about the design decision as of yet - But this can be changed after.

Reply:
I am not sure if the current format used to return data from rsync/docker status is ideal. The original plan was to return a string describing if there was any issue during the check and in case of problems also return the error message. Probably there is a better approach to notify errors during actor execution, without mentioning the point if rsync being not available should be reported as an error or just the correct result of a fact checking. Anyway, current implementation is similar with what we had for 0.1, so I am assuming the code as "correct" and probably changing the schema is the best solution.

We should probably apply @vinzenz suggestion both for rsync and docker at tragetinfo schema.
```
diff --git a/examples/schema/targetinfo.py b/examples/schema/targetinfo.py
index 70a63fc..4e02b2f 100644
--- a/examples/schema/targetinfo.py
+++ b/examples/schema/targetinfo.py
@@ -2,8 +2,14 @@ import jsl
 
 
 class TargetInfo(jsl.Document):
-    docker = jsl.ArrayField(jsl.StringField(), max_items=2)
-    rsync = jsl.ArrayField(jsl.StringField(), max_items=2)
+    docker = jsl.ArrayField([
+        jsl.StringField(),
+        jsl.OneOfField([jsl.StringField(), jsl.NullField()])
+    ])
+    rsync = jsl.ArrayField([
+        jsl.StringField(),
+        jsl.OneOfField([jsl.StringField(), jsl.NullField()])
+    ])
     containers = jsl.ArrayField([
         jsl.StringField(),
         jsl.ArrayField(jsl.StringField())
```

Reply:
@vinzenz About the "containers" key, containers list actor will list the name of all existing containers, both from "docker ps" and from leapp macro containers path. We can have cases where docker fails, but actor is still able to find containers at leapp macro containers path. That is why even with an error we are returning the containers list result. On the same line as previous comment this doesn't look ideal and probably any new approach used for rsync/docker can be replicated here, in a sense of not returning a string to determine a error/success status

Reply:
I committed this version. Regarding containers, if it still can return partial data in case of failure of the docker command, we should split it into two actors, one for docker, other for the filesystem path.